src/rules/rename-default-import.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @fileoverview Rule to enforce aliases for default imports
+ * @author Michał Kołodziejski
+ */
+
+import docsUrl from '../docsUrl'
+
+
+function isDefaultImport(specifier) {
+  if (specifier.type === 'ImportDefaultSpecifier') {
+    return true
+  } else if (specifier.type === 'ImportSpecifier' && specifier.imported.name === 'default') {
+    return true
+  } else {
+    return false
+  }
+}
+
+
+function handleImport(context, specifier, source) {
+  const {value: packageName} = source
+  const {local: {name: importAlias}} = specifier
+  const mappings = context.options[0]
+
+  if (Object.keys(mappings).indexOf(packageName) === -1) {
+    return
+  }
+
+  if (mappings[packageName] !== importAlias) {
+    context.report({
+      node: source,
+      message: `Default import from '${packageName}' should be aliased to `
+               + `${mappings[packageName]}, not ${importAlias}`,
+      fix: fixer => {
+        let newAlias = mappings[packageName]
+        if (specifier.imported && specifier.imported.name === 'default') {
+          newAlias = `default as ${mappings[packageName]}`
+        }
+
+        return fixer.replaceText(specifier, newAlias)
+      },
+    })
+
+    const declaredVariable = context.getDeclaredVariables(specifier)[0]
+    for (const variableReference of declaredVariable.references) {
+      context.report({
+        node: variableReference.identifier,
+        message: `Using incorrect binding name '${variableReference.identifier.name}' `
+                 + `instead of ${mappings[packageName]} for `
+                 + `default import from package ${packageName}`,
+        fix: fixer => {
+          return fixer.replaceText(variableReference.identifier, mappings[packageName])
+        },
+      })
+    }
+  }
+}
+
+
+module.exports = {
+  meta: {
+      docs: {
```

Comment:
too much indent; the rest of the file is using 4 spaces (also below inside `create`)